docs/spec/spec.md

```diff
@@ -185,15 +185,15 @@ public function main() returns error? {
 ```
 
 
-## 4.2. Server Streaming RPC
+## 4.2. Server streaming RPC
```

Comment:
```suggestion
## 4.2. Server streaming RPC
```
Should this be `RPC server streaming`?

Reply:
No, `Server streaming RPC` is the correct term.